web/themes/joinup/joinup_theme.theme

```diff
@@ -116,6 +146,12 @@ function joinup_theme_preprocess_page(&$variables) {
     }
   }
 
+  if ($is_form_container) {
+    $variables['container_class'] = 'content-container--form';
```

Comment:
There is content-container--default class, so I think that it would be consistent to have this class in markup even if it is not used in css.